src/DrupalSetup.php

```diff
@@ -9,63 +9,66 @@
 
 trait DrupalSetup
 {
-  /**
-   * Entities to clean up.
-   *
-   * @var \Drupal\Core\Entity\EntityInterface[]
-   */
-  protected $cleanupEntities = [];
+    /**
+     * Entities to clean up.
+     *
+     * @var \Drupal\Core\Entity\EntityInterface[]
+     */
+    protected $cleanupEntities = [];
 
-  /**
-   * Bootstrap Drupal. Call this from your setUp() method.
-   */
-  public function setupDrupal()
-  {
-    // Bootstrap Drupal so we can use Drupal's built in functions.
-    $finder = new DrupalFinder();
-    $finder->locateRoot(getcwd());
-    $classLoader = include $finder->getVendorDir() . '/autoload.php';
-    $base_url = getenv('DTT_BASE_URL');
-    $parsed_url = parse_url($base_url);
-    $server = [
-      'SCRIPT_FILENAME' => getcwd() . '/index.php',
-      'SCRIPT_NAME' => isset($parsed_url['path']) ? $parsed_url['path'] . 'index.php' : '/index.php',
-    ];
-    $request = Request::create($base_url, 'GET', [], [], [], $server);
-    $kernel = DrupalKernel::createFromRequest($request, $classLoader, 'existing-site-testcase', false, $finder->getDrupalRoot());
-    chdir(DRUPAL_ROOT);
-    $kernel->prepareLegacyRequest($request);
+    /**
+     * Bootstrap Drupal. Call this from your setUp() method.
+     */
+    public function setupDrupal()
+    {
+      // Bootstrap Drupal so we can use Drupal's built in functions.
```

Comment:
The comment indentation is different to the code? Same below also.

Reply:
That's really bizarre that phpcs doesn't catch that as part of psr2. I've fixed that for now manually.